COPY/usr/bin/manageiq-db-ready

```diff
@@ -25,8 +27,11 @@ end
 def manageiq_db_ready?(db_yaml, env = "production")
   host, port, dbname, password = db_yaml[env].values_at("host", "port", "database", "password")
   password = ManageIQ::Password.try_decrypt(password)
+  host_arg = host.nil?     ? "" : "--host=#{host} "         # If not present, let postgres decide to use localhost/socket
+  port_arg = port.nil?     ? "" : "--port=#{port} "
+  pass_arg = password.nil? ? "" : "PGPASSWORD=#{password} "
 
-  results = `PGPASSWORD=#{password} psql --host=#{host} --port=#{port || 5432} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
+  results = `#{pass_arg}psql #{host_arg}#{port_arg} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

Comment:
The spaces is weird, but otherwise, this is fine.  Wonder if it's just cleaner to do:

```suggestion
  results = `#{pass_arg} psql #{host_arg} #{port_arg} --dbname=#{dbname} --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"`
```

And then just don't have trailing spaces above.

Reply:
Ugh, then you have 
```
 psql  --dbname...
^    ^   <==== spaces here
``` 
in the default case with no password, host, port in the database.yml. That's why I put it the way it is.

Reply:
does that matter? extra spaces in the command are ignored
```
[root@manageiq ~]# PGPASSWORD=smartvm psql       --user=root --host=localhost --port=5432 --dbname=vmdb_production --tuples-only --no-align --command="SELECT COUNT(*) FROM miq_regions"
1
```

Reply:
Nope, but the leading space pains my soul. 🤣

You never know who's going to want to parse your awful leading space string:  https://github.com/ManageIQ/pg-dsn_parser/pull/16/files

Reply:
.strip lol

Reply:
I'm not really concerned either way...for me, in general, I prefer code readability over any intermediate thing that probably isn't seen by a human and works either way, which is why I brought it up.

Reply:
🤣 Ok, I pushed the changes.